src/file.ts

```diff
@@ -449,8 +433,10 @@ export class PreloadFile<FS extends FileSystem> extends File<FS> {
 		if (end > this.stats.size) {
 			this.stats.size = end;
 			if (end > this._buffer.byteLength) {
-				if (this._buffer.buffer.resizable && this._buffer.buffer.maxByteLength! <= end) {
-					this._buffer.buffer.resize(end);
+				if ((this._buffer.buffer as ArrayBuffer).resizable && this._buffer.buffer.maxByteLength! <= end) {
```

Comment:
If possible, I'd like to avoid the casting here, and perform a type-safe check instead.